default_payment_config.xml

```diff
@@ -1,4 +1,14 @@
 <?xml version="1.0"?>
+<!--
+/**
```

Comment:
Minor thing, just FYI:

Since those are XML files, you could remove the PHP comment formatting entirely (`/**` ... `*/`), and leave it like this:

```
<!--
Shop System Plugins:
- Terms of Use can be found under:
https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
- License can be found under:
https://github.com/wirecard/oxid-ee/blob/master/LICENSE
-->
```